### PR TITLE
enabled webinterface for localhost

### PR DESCRIPTION
**Pull request by akhon**

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -12,9 +12,11 @@ monit_mail_enabled: false
 monit_mailserver_host: localhost
 monit_mailserver_port: 25
 
-monit_webinterface_enabled: false
+monit_webinterface_enabled: true
 monit_webinterface_bind: 0.0.0.0
```
Comment (max-lobur) [inline defaults/main.yml:16]: maybe bind to 127.0.0.1 to be sure?

Comment (akhon) [reply defaults/main.yml:16]: allow only for localhost.

Comment (akhon) [reply defaults/main.yml:16]: plus, it's not opened in security groups. so it's safe.

Comment (akhon) [reply defaults/main.yml:16]: it's required for monit control, that's why we need it.
we can add telegraf on top of it later on

**Comment by akhon** [comment]
`monit status` example:
```
The Monit daemon 5.6 uptime: 5m

Process 'telegraf'
  status                            Running
  monitoring status                 Monitored
  pid                               20265
  parent pid                        1
  uptime                            2m
  children                          0
  memory kilobytes                  48304
  memory kilobytes total            48304
  memory percent                    1.2%
  memory percent total              1.2%
  cpu percent                       0.0%
  cpu percent total                 0.0%
  data collected                    Thu, 23 May 2019 12:44:34

Process 'td-agent'
  status                            Running
  monitoring status                 Monitored
  pid                               20398
  parent pid                        1
  uptime                            2m
  children                          1
  memory kilobytes                  24052
  memory kilobytes total            136344
  memory percent                    0.6%
  memory percent total              3.5%
  cpu percent                       0.0%
  cpu percent total                 0.0%
  data collected                    Thu, 23 May 2019 12:44:34

System 'forwarder_system_monitor'
  status                            Running
  monitoring status                 Monitored
  load average                      [0.16] [0.72] [0.63]
  cpu                               0.0%us 0.0%sy 0.0%wa
  memory usage                      317604 kB [8.2%]
  swap usage                        0 kB [0.0%]
  data collected                    Thu, 23 May 2019 12:44:34

Filesystem 'rootfs'
  status                            Accessible
  monitoring status                 Monitored
  permission                        775
  uid                               1000
  gid                               1000
  filesystem flags                  0x1000
  block size                        4096 B
  blocks total                      2028792 [7925.0 MB]
  blocks free for non superuser     1314101 [5133.2 MB] [64.8%]
  blocks free total                 1422920 [5558.3 MB] [70.1%]
  inodes total                      524288
  inodes free                       369002 [70.4%]
  data collected                    Thu, 23 May 2019 12:44:34
```

**Pull request merged**